scripts/aggregate_cells.py

```diff
@@ -94,29 +94,37 @@ def aggregate(
 
 if __name__ == "__main__":
 
-    # transforming snakemake objects into python standard datatypes
-    sqlfiles = [str(sqlfile) for sqlfile in snakemake.input["sql_files"]]
-    cell_count_out = [str(out_name) for out_name in snakemake.output["cell_counts"]]
-    aggregate_profile_out = [
-        str(out_name) for out_name in snakemake.output["aggregate_profile"]
-    ]
-    meta_data_dir = itertools.repeat(str(snakemake.input["metadata"]))
-    barcode_map = itertools.repeat(str(snakemake.input["barcodes"]))
-    config_path = itertools.repeat(str(snakemake.params["aggregate_config"]))
-    inputs = list(
-        zip(sqlfiles, meta_data_dir, barcode_map, cell_count_out, aggregate_profile_out, config_path)
+    # CLI arguments
```

Comment:
One thing I like doing with argparse (or other CLI) is to abstract it into a different function that you import.

Essentially, you can write a function called `grab_args()`, import it here, and then call it: `args = grab_args()`.

then reference them like you do below (e.g. `sql_file = args.input`).

Does this make sense? It has a huge benefit of only needing to maintain one argument parser, and making each `.py` function much more digestible.

Reply:
I do not thing using `argparse` will be necessary anymore. It seems that the `script` rule attribute does support multiprocessing. 

https://github.com/WayScience/CytoPipe/pull/11#discussion_r916287016